File: __Viejo/Formularios/muestra_texto.vb
Public Class muestra_texto
    Private str As String
    Public Sub New()

        ' Esta llamada es exigida por el diseñador.
        InitializeComponent()

        ' Agregue cualquier inicialización después de la llamada a InitializeComponent().

    End Sub

    Public Sub New(ByVal _str As String)

        ' Esta llamada es exigida por el diseñador.
        InitializeComponent()

        ' Agregue cualquier inicialización después de la llamada a InitializeComponent().
        str = _str
    End Sub

    Private Sub muestra_texto_Load(sender As Object, e As EventArgs) Handles MyBase.Load
        txt_msg.Text = str
    End Sub

    Private Sub cmd_ok_Click(sender As Object, e As EventArgs) Handles cmd_ok.Click
        Me.Close()
    End Sub
End Class

File: Formularios/muestra_texto.cs
using System;

namespace Centrex
{
    public partial class muestra_texto
    {
        private string str;
        public muestra_texto()
        {

            // Esta llamada es exigida por el diseñador.
            InitializeComponent();

            // Agregue cualquier inicialización después de la llamada a InitializeComponent().

        }

        public muestra_texto(string _str)
        {

            // Esta llamada es exigida por el diseñador.
            InitializeComponent();

            // Agregue cualquier inicialización después de la llamada a InitializeComponent().
            str = _str;
        }

        private void muestra_texto_Load(object sender, EventArgs e)
        {
            txt_msg.Text = str;
        }

        private void cmd_ok_Click(object sender, EventArgs e)
        {
            Close();
        }
    }
}
